### ISSUE TITLE
Consider removing detailed explanation of CLI options

### ISSUE THREAD

**Issue by geigerzaehler** [radicle-dev/radicle-run#23]
We’re documenting in great detail the options and capabilities of `radicle-registry-cli`. In a [few places](https://github.com/radicle-dev/radicle-run/blob/master/docs/submit-transactions.md#command-line-options) we even repeat the `--help` output of the command. With this we risk the docs getting out of sync and we create an overhead for updating the CLI. We should evaluate whether the repeated documentation provides enough value to outweigh these drawbacks. If not we should consider simplifying the docs.

**Comment by geigerzaehler** [comment]
In my opinion listing all possible transaction and the command line options does not enough vaue and should be removed from the [“Submit transaction” guide](https://github.com/radicle-dev/radicle-run/blob/master/docs/submit-transactions.md#command-line-options). In the long run we should add man pages to our binaries so that we can document everything in there.

**Comment by NunoAlexandre** [comment]
I agree with this. that's something I reported when writing those bits even.

I reiterate my suggestion to host the CLI docs here. Then we don't require users to go to the terminal and run help commands for each (sub) command and we provide the docs in a way that is generated and not duplicated.

What do you think?

**Comment by geigerzaehler** [comment]
> I reiterate my suggestion to host the CLI docs here.

Could you elaborate on how you imagine this to work?

To be clear: I think the value that replicating the documentation has is that _potential_ users (especially developers) get a good grasp of what the CLI can do for them _before_ they download it. The value for users that have the CLI is only marginal. It’s more likely that they explore the help text first and only move to docs on the web if the help does not answer their question.
